tonic_validate/metrics/answer_match_metric.py

```diff
@@ -0,0 +1,21 @@
+import logging
+
+from tonic_validate.classes.llm_response import LLMResponse
+from tonic_validate.metrics.binary_metric import BinaryMetric
+from tonic_validate.services.openai_service import OpenAIService
+
+logger = logging.getLogger()
+
+
+class AnswerMatchMetric(BinaryMetric):
+    def __init__(self, name: str, answer: str, case_sensitive: bool = False):
```

Comment:
I think the `name` parameter for each of these metrics should have a default value (like "answer_match_metric" in this case) so the user doesn't necessarily have to name the metric.

Reply:
I kind of like forcing a name because I could easily see someone using multiple answer match metrics and then getting confused which one is which in the UI. I had considered just generating a name, but I think that's impractical because someone would likely be trying to match a very long piece of text and we'd need to truncate it to the point where it's not useful

Reply:
Ok, that's fine. Let's get some doc strings in their that explain what the name is and that it will show up in the UI when the metric is logged.